pkg/operator/ceph/file/filesystem.go

```diff
@@ -51,37 +49,31 @@ func createFilesystem(
 	ownerInfo *k8sutil.OwnerInfo,
 	dataDirHostPath string,
 ) error {
+	logger.Infof("start running mdses for filesystem %q", fs.Name)
```

Comment:
In the past, starting RGW first caused Ceph to create the pools. We don't have that same behavior with mds, right? Just wanted to confirm that Rook still owns pool creation.

Reply:
Correct, pools are not auto-created if they don't exist AFAIK.